src/types/OphanTypes.ts

```diff
@@ -2,14 +2,16 @@ export type OphanProduct =
     | 'CONTRIBUTION'
     | 'MEMBERSHIP_SUPPORTER'
     | 'DIGITAL_SUBSCRIPTION'
-    | 'PRINT_SUBSCRIPTION';
+    | 'PRINT_SUBSCRIPTION'
+    | 'PUZZLES_APP';
```

Comment:
We'll have the same issue described above regarding the thrift definitions as one exists for Ophan products: https://github.com/guardian/ophan/blob/69a83e90e421cc610ee28fdcfb1bd8df451f1905/event-model/src/main/thrift/product.thrift

I don't think `product` is a required field in a `componentEvent` so we may not need to pass it here.